CONTRIBUTING.md

```diff
@@ -0,0 +1,209 @@
+* Fix test for scenario.Application - should remove old iframe
+* docs - various doc fixes
+* docs - stripping extra new lines
+* Replaced double line break with single when text is fetched from Google
+* Added support for properties in documentation
+
+All of these messages try to specify where is the change. But they don’t share any convention...
+
+Look at these messages:
+
+* fix comment stripping
+* fixing broken links
+* Bit of refactoring
+* Check whether links do exist and throw exception
+* Fix sitemap include (to work on case sensitive linux)
+
+Are you able to guess what’s inside ? These messages miss place specification...
+So maybe something like parts of the code: docs, docs-parser, compiler, scenario-runner, …
+
+I know, you can find this information by checking which files had been changed, but that’s slow. And when looking in git
+history I can see all of us tries to specify the place, only missing the convention.
+
+---
+
+Format of the commit message
+----------------------------
+
+```
+[#<issue1>] [#<issue2>] <type>(<scope>): <subject>
+```
+
+A commit message should contain one line and should not be longer than 100 characters. This allows the message to be
+easier to read on github as well as in various git tools.
+
+### Referencing issues
+
+Related issue(s), if any, should be listed in the beginning of a commit message.
+
+### Allowed `<type>`
+
+This type should be assigned from the perspective of a customer, since these types end up in the changelog.
+
+* feat (feature)
+* fix (bug fix)
+* docs (documentation)
+* style (formatting, missing semi colons, …)
+* refactor
+* test (when adding missing tests)
+* chore (CI, builds, maintainance, …)
+
+### Allowed `<scope>`
+
+Scope could be anything specifying place of the commit change.
+
+Use a microservice name (or comma-separated microservice names, or a corresponding folder name like `deployment`
+, `commons` or `scripts`) as a scope.
+
+Undefined (omitted) scope means that commit changes a lot of components at once (although ideally commits and PRs should
+not be huge).
+
+### `<subject>`
+
+Subject line contains succinct description of the change.
+
+* use imperative, present tense: “change” not “changed” nor “changes”
+* don't capitalize first letter
+* no dot (.) at the end
+
+Examples
+--------
+
+```
+#123 feat: anonymize tattoos and moles
+```
+
+```
+#456 fix(face_generator): remove unnecessary highlights from faces
+```
+
+```
+chore(deployment): make a new Docker image 
+```
+
+# Python Code Style Reference
+## Tests
+All new features and bug fixes must be tested. 
+If you add a new feature, please add unit and functional tests.
+If you fix a bug, add tests that check if it was fixed. 
+
+A "Best of the Best Practices" [(BOBP) guide to developing in Python](https://gist.github.com/sloria/7001839).
+
+## Formatting
+
+To format code we use [black](https://github.com/psf/black).
+
+Use one of the following commands to format your changes before committing them:
```

Comment:
Maybe also mention that the version of black is important here? Since with different versions the code can be formatted differently.

Reply:
Added a note about it